src/exampleTypes/Facet.js

```diff
@@ -135,29 +140,57 @@ let Facet = injectTreeNode(
             </label>
           )
         }, _.get('context.options', node))}
-        <Flex
-          className="contexture-facet-cardinality"
-          style={{ justifyContent: 'space-between' }}
-        >
-          {!!node.context.cardinality && (
+        {!!node.context.cardinality && (
+          <Flex
+            className="contexture-facet-cardinality"
+            justifyContent="space-between"
+          >
             <div>
-              Showing {_.min([node.size || 10, node.context.options.length])} of{' '}
+              Showing{' '}
+              {_.min([node.size || minSize, node.context.options.length])} of{' '}
               {node.context.cardinality}
             </div>
-          )}
-          {node.context.cardinality > (node.size || 10) && (
             <div>
-              <a
-                onClick={() =>
-                  tree.mutate(node.path, { size: (node.size || 10) + 10 })
-                }
-                style={{ cursor: 'pointer' }}
-              >
-                View More
-              </a>
+              {_.flow(
+                _.compact,
+                F.intersperse(' — ')
+              )([
+                _.min([node.size, node.context.options.length]) > minSize && (
+                  <a
+                    key="less"
+                    onClick={() =>
+                      tree.mutate(node.path, {
+                        size: _.max([
+                          _.min([
+                            node.size,
+                            ceilTens(node.context.options.length),
```

Comment:
Just following up on our conversation - this is clever, but doesn't work if you've customized the interval.

Also, lets factor this out into a more reusable "clamp" thing since this logic might be usable in other places (like term stats)

Reply:
something like this: https://runkit.com/daedalus28/clampmodel